ballerina-tests/tests/45_services_with_headers_service.bal

```diff
@@ -83,10 +83,10 @@ service "HeadersService" on ep45 {
     remote function bidirectionalStr(HeadersServiceHSResCaller caller, ContextHSReqStream req) returns error? {
         map<string|string[]> responseHeaders = {};
         if !req.headers.hasKey("bidi-steaming-req-header") {
-            check caller->sendError(error grpc:AbortedError("bidi-steaming-req-header header is missing"));
+            checkpanic caller->sendError(error grpc:AbortedError("bidi-steaming-req-header header is missing"));
             return;
         } else {
-            string headerValue = check grpc:getHeader(req.headers, "bidi-steaming-req-header");
+            string headerValue = checkpanic grpc:getHeader(req.headers, "bidi-steaming-req-header");
```

Comment:
```suggestion
            string headerValue = check grpc:getHeader(req.headers, "bidi-steaming-req-header");
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd